### ISSUE TITLE
feat(alerting): Overriding telegram id and token

### ISSUE THREAD

**Issue by sbriskin** [TwiN/gatus#717]
### Describe the feature request

Add support for overriding telegram recipients and bots. Something like this:
```
alerting:
  telegram:
    token: "NNNNNNNNN:xxxxxxx_yyyyyyyyy"
    id: "123456789"
    overrides:
      - group: "A"
        id: "-1009876543210"
      - group: "B"
        id: "-1001234567890"
        token: "NNNNNNNNN:aaaaaaaaa_bbbbbbbb"
```

### Why do you personally want this feature to be implemented?

This is amazing app, but only one telegram id limit makes me cry.
I have a few telegram chats: one for devops team with their ci resources, one for QA with their test servers and one only for myself for other tasks. So I have to manually forward notifications from my personal chat to others. Please add override rules for telegram.

**Comment by TwiN** [comment]
If anybody that wants to pick this up, here's 2 examples of the work needed: #272 #292

**Comment by lefes** [comment]
@TwiN Thank you for providing these examples. They were helpful for me in preparing the PR. I hope it passes your review. If needed, I will make any necessary fixes promptly and stay in touch.

**Comment by lefes** [comment]
@TwiN, I believe this issue can be closed. Would you agree?

**Comment by TwiN** [comment]
Yep, you're right @lefes!

Resolved by #826
